### DISCUSSION TITLE
Getting started query

### DISCUSSION THREAD

**Discussion by ripjal** [open-telemetry/opentelemetry-java-instrumentation#4353] [Q&A]
I have been trying to get start with open telemetry for my java web application project.
I am using `java -javaagent:path/to/opentelemetry-javaagent-all.jar \
     -Dotel.resource.attributes=service.name=your-service-name \
     -Dotel.traces.exporter=zipkin \
     -jar myapp.jar` to export traces to the zip backend successfully.

My question is that when I use this agent ,does it run collector as well ?So I believe it instruments and sends all telemetry data to default port and then have collector running and receiving data from this default port and then exporting to zipkin as well  ?Is My understanding correct ?

**Comment by mateuszrzeszutek** [comment]
Hey @ripjal ,

> My question is that when I use this agent ,does it run collector as well ?

No, this project does not include the [collector](https://github.com/open-telemetry/opentelemetry-collector/). You'll have to set it up separately using instructions from the collector repo.

> So I believe it instruments and sends all telemetry data to default port and then have collector running and receiving data from this default port and then exporting to zipkin as well ?

By default the zipkin exporter sends data to the `http://localhost:9411/api/v2/spans` endpoint (see [the docs](https://github.com/open-telemetry/opentelemetry-java/tree/main/sdk-extensions/autoconfigure#zipkin-exporter)). It is the default port in a locally-deployed OpenTelemetry collector, but if you have a local zipkin backend deployed it'll also work (locally).
The collector can be configured to send traces to zipkin - please check out the [collector contrib](https://github.com/open-telemetry/opentelemetry-collector-contrib) repo (zipkin receiver/exporter does not seem to be bundled in the core distribution) and the [collector docs](https://opentelemetry.io/docs/collector/configuration/).

**Reply by ripjal** [reply]
Thanks @mateuszrzeszutek  for your reply.I am just using this javaagent and data is being exported to zipkin..Isnt collector required here?
I am not getting when exactly collector need to be run and where do I get its jar file to run

**Reply by iNikem** [reply]
@ripjal Have you read https://opentelemetry.io/docs/collector/getting-started/ ?
